<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorTransformWizardPageForm</class>
 <widget class="QWizardPage" name="ColorTransformWizardPageForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color Transform</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_16">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Type</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_15">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_14">
        <item row="0" column="0">
         <widget class="QComboBox" name="m_colorTransformTypeComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QStackedWidget" name="m_stackedWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>RGB - IHS</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_17">
               <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayout_13">
                 <item row="0" column="0">
                  <layout class="QGridLayout" name="gridLayout_12">
                   <item row="0" column="0">
                    <layout class="QGridLayout" name="gridLayout_10">
                     <item row="0" column="0">
                      <layout class="QGridLayout" name="gridLayout_4">
                       <item row="0" column="0">
                        <layout class="QGridLayout" name="gridLayout">
                         <item row="0" column="0">
                          <widget class="QLabel" name="label">
                           <property name="text">
                            <string>Red</string>
                           </property>
                           <property name="alignment">
                            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                           </property>
                          </widget>
                         </item>
                         <item row="1" column="0">
                          <widget class="QLabel" name="label_3">
                           <property name="text">
                            <string>Green</string>
                           </property>
                           <property name="alignment">
                            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                           </property>
                          </widget>
                         </item>
                         <item row="2" column="0">
                          <widget class="QLabel" name="label_5">
                           <property name="text">
                            <string>Blue</string>
                           </property>
                           <property name="alignment">
                            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </item>
                       <item row="0" column="1">
                        <layout class="QGridLayout" name="gridLayout_2">
                         <item row="0" column="0">
                          <widget class="QComboBox" name="m_rgbRComboBox">
                           <property name="sizePolicy">
                            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                             <horstretch>0</horstretch>
                             <verstretch>0</verstretch>
                            </sizepolicy>
                           </property>
                          </widget>
                         </item>
                         <item row="1" column="0">
                          <widget class="QComboBox" name="m_rgbGComboBox">
                           <property name="sizePolicy">
                            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                             <horstretch>0</horstretch>
                             <verstretch>0</verstretch>
                            </sizepolicy>
                           </property>
                          </widget>
                         </item>
                         <item row="2" column="0">
                          <widget class="QComboBox" name="m_rgbBComboBox">
                           <property name="sizePolicy">
                            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                             <horstretch>0</horstretch>
                             <verstretch>0</verstretch>
                            </sizepolicy>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </item>
                      </layout>
                     </item>
                     <item row="0" column="1">
                      <layout class="QGridLayout" name="gridLayout_9">
                       <item row="0" column="0">
                        <layout class="QGridLayout" name="gridLayout_7">
                         <item row="0" column="0">
                          <widget class="QLabel" name="label_2">
                           <property name="text">
                            <string>Band</string>
                           </property>
                          </widget>
                         </item>
                         <item row="1" column="0">
                          <widget class="QLabel" name="label_4">
                           <property name="text">
                            <string>Band</string>
                           </property>
                          </widget>
                         </item>
                         <item row="2" column="0">
                          <widget class="QLabel" name="label_6">
                           <property name="text">
                            <string>Band</string>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </item>
                       <item row="0" column="1">
                        <layout class="QGridLayout" name="gridLayout_8">
                         <item row="0" column="0">
                          <widget class="QComboBox" name="m_rgbRBandComboBox">
                           <property name="maximumSize">
                            <size>
                             <width>50</width>
                             <height>16777215</height>
                            </size>
                           </property>
                          </widget>
                         </item>
                         <item row="1" column="0">
                          <widget class="QComboBox" name="m_rgbGBandComboBox">
                           <property name="maximumSize">
                            <size>
                             <width>50</width>
                             <height>16777215</height>
                            </size>
                           </property>
                          </widget>
                         </item>
                         <item row="2" column="0">
                          <widget class="QComboBox" name="m_rgbBBandComboBox">
                           <property name="maximumSize">
                            <size>
                             <width>50</width>
                             <height>16777215</height>
                            </size>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                   <item row="1" column="0">
                    <widget class="Line" name="line">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="0">
                    <layout class="QGridLayout" name="gridLayout_11">
                     <item row="0" column="0">
                      <layout class="QGridLayout" name="gridLayout_5">
                       <item row="0" column="0">
                        <widget class="QLabel" name="label_7">
                         <property name="text">
                          <string>Minimum RGB Value</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="0">
                        <widget class="QLabel" name="label_8">
                         <property name="text">
                          <string>Maximum RGB Value</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item row="0" column="1">
                      <layout class="QGridLayout" name="gridLayout_6">
                       <item row="0" column="0">
                        <widget class="QLineEdit" name="m_rgbMinLineEdit">
                         <property name="text">
                          <string>0</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="0">
                        <widget class="QLineEdit" name="m_rgbMaxLineEdit">
                         <property name="text">
                          <string>255</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                 <item row="1" column="0">
                  <spacer name="verticalSpacer">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_5">
           <layout class="QGridLayout" name="gridLayout_29">
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox_7">
              <property name="title">
               <string>IHS - RGB</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_31">
               <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayout_30">
                 <item row="0" column="0">
                  <layout class="QGridLayout" name="gridLayout_18">
                   <item row="0" column="0">
                    <layout class="QGridLayout" name="gridLayout_19">
                     <item row="0" column="0">
                      <layout class="QGridLayout" name="gridLayout_20">
                       <item row="0" column="0">
                        <layout class="QGridLayout" name="gridLayout_21">
                         <item row="0" column="0">
                          <widget class="QLabel" name="label_9">
                           <property name="text">
                            <string>Intensity</string>
                           </property>
                           <property name="alignment">
                            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                           </property>
                          </widget>
                         </item>
                         <item row="1" column="0">
                          <widget class="QLabel" name="label_10">
                           <property name="text">
                            <string>Hue</string>
                           </property>
                           <property name="alignment">
                            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                           </property>
                          </widget>
                         </item>
                         <item row="2" column="0">
                          <widget class="QLabel" name="label_11">
                           <property name="text">
                            <string>Saturation</string>
                           </property>
                           <property name="alignment">
                            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </item>
                       <item row="0" column="1">
                        <layout class="QGridLayout" name="gridLayout_22">
                         <item row="0" column="0">
                          <widget class="QComboBox" name="m_ihsIComboBox">
                           <property name="sizePolicy">
                            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                             <horstretch>0</horstretch>
                             <verstretch>0</verstretch>
                            </sizepolicy>
                           </property>
                          </widget>
                         </item>
                         <item row="1" column="0">
                          <widget class="QComboBox" name="m_ihsHComboBox">
                           <property name="sizePolicy">
                            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                             <horstretch>0</horstretch>
                             <verstretch>0</verstretch>
                            </sizepolicy>
                           </property>
                          </widget>
                         </item>
                         <item row="2" column="0">
                          <widget class="QComboBox" name="m_ihsSComboBox">
                           <property name="sizePolicy">
                            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                             <horstretch>0</horstretch>
                             <verstretch>0</verstretch>
                            </sizepolicy>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </item>
                      </layout>
                     </item>
                     <item row="0" column="1">
                      <layout class="QGridLayout" name="gridLayout_23">
                       <item row="0" column="0">
                        <layout class="QGridLayout" name="gridLayout_24">
                         <item row="0" column="0">
                          <widget class="QLabel" name="label_12">
                           <property name="text">
                            <string>Band</string>
                           </property>
                          </widget>
                         </item>
                         <item row="1" column="0">
                          <widget class="QLabel" name="label_13">
                           <property name="text">
                            <string>Band</string>
                           </property>
                          </widget>
                         </item>
                         <item row="2" column="0">
                          <widget class="QLabel" name="label_14">
                           <property name="text">
                            <string>Band</string>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </item>
                       <item row="0" column="1">
                        <layout class="QGridLayout" name="gridLayout_25">
                         <item row="0" column="0">
                          <widget class="QComboBox" name="m_ihsIBandComboBox">
                           <property name="maximumSize">
                            <size>
                             <width>50</width>
                             <height>16777215</height>
                            </size>
                           </property>
                          </widget>
                         </item>
                         <item row="1" column="0">
                          <widget class="QComboBox" name="m_ihsHBandComboBox">
                           <property name="maximumSize">
                            <size>
                             <width>50</width>
                             <height>16777215</height>
                            </size>
                           </property>
                          </widget>
                         </item>
                         <item row="2" column="0">
                          <widget class="QComboBox" name="m_ihsSBandComboBox">
                           <property name="maximumSize">
                            <size>
                             <width>50</width>
                             <height>16777215</height>
                            </size>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                   <item row="1" column="0">
                    <widget class="Line" name="line_2">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="0">
                    <layout class="QGridLayout" name="gridLayout_26">
                     <item row="0" column="0">
                      <layout class="QGridLayout" name="gridLayout_27">
                       <item row="0" column="0">
                        <widget class="QLabel" name="label_15">
                         <property name="text">
                          <string>Minimum RGB Value</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="0">
                        <widget class="QLabel" name="label_16">
                         <property name="text">
                          <string>Maximum RGB Value</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item row="0" column="1">
                      <layout class="QGridLayout" name="gridLayout_28">
                       <item row="0" column="0">
                        <widget class="QLineEdit" name="m_ihsMinLineEdit">
                         <property name="text">
                          <string>0</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                         </property>
                        </widget>
                       </item>
                       <item row="1" column="0">
                        <widget class="QLineEdit" name="m_ihsMaxLineEdit">
                         <property name="text">
                          <string>255</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                 <item row="1" column="0">
                  <spacer name="verticalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_colorTransformTypeComboBox</tabstop>
  <tabstop>m_rgbRComboBox</tabstop>
  <tabstop>m_rgbRBandComboBox</tabstop>
  <tabstop>m_rgbGComboBox</tabstop>
  <tabstop>m_rgbGBandComboBox</tabstop>
  <tabstop>m_rgbBComboBox</tabstop>
  <tabstop>m_rgbBBandComboBox</tabstop>
  <tabstop>m_rgbMinLineEdit</tabstop>
  <tabstop>m_rgbMaxLineEdit</tabstop>
  <tabstop>m_ihsIComboBox</tabstop>
  <tabstop>m_ihsIBandComboBox</tabstop>
  <tabstop>m_ihsHComboBox</tabstop>
  <tabstop>m_ihsHBandComboBox</tabstop>
  <tabstop>m_ihsSComboBox</tabstop>
  <tabstop>m_ihsSBandComboBox</tabstop>
  <tabstop>m_ihsMinLineEdit</tabstop>
  <tabstop>m_ihsMaxLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_colorTransformTypeComboBox</sender>
   <signal>activated(int)</signal>
   <receiver>m_stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
